analyzers/tests/SonarAnalyzer.Test/TestCases/Razor/EmptyProject/EmptyProject.csproj

```diff
@@ -1,11 +1,167 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Made this a .NET FM project, to test a bit more realistically.

I think a better way is by attaching a debugger, especially in the initial stages, where we do not want to find the UT framework.